src/App.vue

```diff
@@ -20,31 +20,39 @@ import {
   watch,
   watchEffect
 } from 'vue'
-import BlockUI from 'primevue/blockui'
-import ProgressSpinner from 'primevue/progressspinner'
-import QueueSidebarTab from '@/components/sidebar/tabs/QueueSidebarTab.vue'
-import { app } from './scripts/app'
-import { useSettingStore } from './stores/settingStore'
+import config from '@/config'
+import { app } from '@/scripts/app'
+import { useSettingStore } from '@/stores/settingStore'
 import { useI18n } from 'vue-i18n'
-import { useWorkspaceStore } from './stores/workspaceStateStore'
-import NodeLibrarySidebarTab from './components/sidebar/tabs/NodeLibrarySidebarTab.vue'
-import GlobalDialog from './components/dialog/GlobalDialog.vue'
-import GlobalToast from './components/toast/GlobalToast.vue'
-import UnloadWindowConfirmDialog from './components/dialog/UnloadWindowConfirmDialog.vue'
-import BrowserTabTitle from './components/BrowserTabTitle.vue'
-import { api } from './scripts/api'
-import { StatusWsMessageStatus } from './types/apiTypes'
-import { useQueuePendingTaskCountStore } from './stores/queueStore'
+import { useWorkspaceStore } from '@/stores/workspaceStateStore'
+import { api } from '@/scripts/api'
+import { StatusWsMessageStatus } from '@/types/apiTypes'
+import { useQueuePendingTaskCountStore } from '@/stores/queueStore'
 import type { ToastMessageOptions } from 'primevue/toast'
 import { useToast } from 'primevue/usetoast'
-import { i18n } from './i18n'
-import { useExecutionStore } from './stores/executionStore'
-import { useWorkflowStore } from './stores/workflowStore'
+import { i18n } from '@/i18n'
+import { useExecutionStore } from '@/stores/executionStore'
+import { useWorkflowStore } from '@/stores/workflowStore'
+import BlockUI from 'primevue/blockui'
+import ProgressSpinner from 'primevue/progressspinner'
+import QueueSidebarTab from '@/components/sidebar/tabs/QueueSidebarTab.vue'
+import NodeLibrarySidebarTab from '@/components/sidebar/tabs/NodeLibrarySidebarTab.vue'
+import GlobalDialog from '@/components/dialog/GlobalDialog.vue'
+import GlobalToast from '@/components/toast/GlobalToast.vue'
+import UnloadWindowConfirmDialog from '@/components/dialog/UnloadWindowConfirmDialog.vue'
+import BrowserTabTitle from '@/components/BrowserTabTitle.vue'
 
 const isLoading = computed<boolean>(() => useWorkspaceStore().spinner)
-const theme = computed<string>(() =>
-  useSettingStore().get('Comfy.ColorPalette')
-)
+
+const { t } = useI18n()
```

Comment:
Please revert some of the moves of stores, as it is an advantage of using composition API is that you can have declaration and usage at the same place. See https://vuejs.org/guide/extras/composition-api-faq.html for details.

![image](https://github.com/user-attachments/assets/3a763026-a9ce-4e64-8d7e-3b3deff64a36)

Reply:
We are loosing time ... I perfectly know what is composition API, doing vue for more than 10 years now. Please merge that PR so I can continue.